src/medications/ViewMedication.tsx

```diff
@@ -297,11 +297,12 @@ const ViewMedication = () => {
           <Row>
             <Column>
               <TextFieldWithLabelFormGroup
+                id="notes"
```

Comment:
```suggestion
                data-testid="notes"
```
these would then be query-able by `.*byTestId()`